### PR TITLE
feat: prepare app to mainnet

### PR DESCRIPTION
**Pull request by pedroferreira1**
- Fix android app name
- Update initial screen
- Change address network byte
- Update server to mainnet
- Update wallet lib version

image

### CODE REVIEW

File: src/screens/InitWallet.js [hunk 1]
```diff
@@ -52,17 +52,13 @@ class WelcomeScreen extends React.Component {
...
-              This wallet is connected to a <Strong>testnet</Strong>.
+              This wallet is connected to the <Strong>mainnet</Strong>.
             </Text>
             <Text style={this.style.text}>
-              This means that{' '}
-              <Strong>your Hathor token (HTR) and any other token may be reset at any time.</Strong>
-            </Text>
-            <Text style={this.style.text}>
-              If someone offers to sell some tokens to you, that person is a scammer.
+              A mobile wallet is not the safest place to store your tokens, so we advise you to have only a small amount of them here.
```
Comment (msbrogli) [inline src/screens/InitWallet.js]: Suggestion: A mobile wallet is not the safest place to store your tokens. So, we advise you to keep only a small amount of tokens here, such as pocket money.

Comment (msbrogli) [reply src/screens/InitWallet.js]: Maybe we should include a page in our website about Token Custody to a longer explanation about how to store tokens [(hardware wallet, software wallet) x (desktop wallet, mobile wallet)]. What do you think?

Comment (obiyankenobi) [reply src/screens/InitWallet.js]: I'd open an issue for this, as we don't have this page yet.

Comment (pedroferreira1) [reply src/screens/InitWallet.js]: I agree with @obiyankenobi that we should do it later, maybe even a blog post would be better than a page on the website. I will create an issue for this.

I've update the text as you suggested.

**Pull request merged**